driver-core/src/main/java/com/datastax/driver/core/ShardingInfo.java

```diff
@@ -25,18 +25,25 @@
   private static final String SCYLLA_PARTITIONER = "SCYLLA_PARTITIONER";
   private static final String SCYLLA_SHARDING_ALGORITHM = "SCYLLA_SHARDING_ALGORITHM";
   private static final String SCYLLA_SHARDING_IGNORE_MSB = "SCYLLA_SHARDING_IGNORE_MSB";
+  private static final String SCYLLA_SHARD_AWARE_PORT = "SCYLLA_SHARD_AWARE_PORT";
```

Comment:
What about SCYLLA_SHARD_AWARE_PORT_SSL? There is a separate shard-aware port for SSL connections.